data/coalition/coalition missions.txt

```diff
@@ -143,3 +143,847 @@ mission "Coalition License Patch"
+			`	As you continue to chat, one of them gets curious and asks you about humanity's population, to which you respond that your species numbers about 100 billion people.`
+			`	"Double that of the Arachi," one of the workers says.`
+			`	"And half that of the Kimek," another one completes.`
+			`	They ask you some more questions about human territory, but avoid getting too technical to the point where it's difficult for you to respond.`
+				goto end
+			label shops
+			`	You try and entertain yourself by looking at the shops here, but pretty much all of them are just filled with the produce of local farms.`
+			`	One of them has a few toys made out of straw. You take a look at the Saryd designs for them. They're nothing particularly special, nor as stereotypical as one might have expected from a species that resembles centaurs. Most of them appear to be replicating creatures of the local fauna or buildings like small houses.`
+			`	You notice two other census workers have come back, so you decide to return to the group and wait for the final member. Some Coalition captains from earlier have gathered together in a bar to lament the long wait.`
+			label end
+			`	After the final Saryd arrives, you all head back to your ship, and you're told that the next destination is <destination>.`
+				accept
+	on visit
+		dialog `You land on <planet>, but realize that your escort carrying the census workers hasn't entered the system yet. Better depart and wait for it.`
+	on complete
+		"coalition jobs" += 2
+		payment 387000
+		conversation
+			`This time the workers mention how they were tasked with gathering data on a part of the planet other than the surroundings of the spaceport, as they pay you <payment>.`
+			`	Thankfully, they mention how the census is nearly over and that this was their final destination, so you won't need to wait for hours in the spaceport like the captains you saw on <origin>.`
+			`	They thank you for helping them do their job, and wish you safe travels.`
+
+
+
+mission "Saryd Couple 1"
+	minor
+	name "Acquiring Funds"
+	description "Inturi and Aunaris, a young Saryd couple, need to make enough money to buy their dream home. Transport Aunaris to <destination>, where she was offered a job in the university there."
+	passengers 1
+	to offer
+		has "Coalition: First Contact: done"
+		random < 60
+	source
+		near "Hunter" 1 2
+	destination "Shadow of Leaves"
+	on offer
+		conversation
+			`While walking around the spaceport, you notice one of the Saryd workers that was refueling your ship is quickly galloping somewhere, only without his uniform, as if his shift just ended. He meets up with another Saryd and starts looking at a map with her.`
+			choice
+				`	(Approach them.)`
+				`	(Go back to your ship and check if it was properly refueled.)`
+					decline
```

Comment:
I think that this choice should lead to some writing where you find that your ship is fine and you decide to move on with your day, instead of just outright ending here.

Reply:
added little bit at the end